packages/types/package.json

```
@@ -48,5 +48,8 @@
         "_ts3.4/*"
       ]
     }
+  },
+  "devDependencies": {
+    "typescript": ">=2.6.0"
```

We use the root package.json to control the TS version installed in the repo.

You can use the `peerDependenciesMeta` to add an optional peer dep on typescript, which we'll need to do to ensure package managers can understand the new requirement

```suggestion
    "typescript": "*"
```